packages/theme/src/ee/store/dashboard/roadmaps.ts

```diff
@@ -63,7 +63,8 @@ export const useDashboardRoadmaps = defineStore("dashboardRoadmaps", () => {
     );
     if (roadmapIdx === -1) return;
 
-    Object.assign(roadmaps.value[roadmapIdx], roadmap);
+    if (roadmaps.value[roadmapIdx])
+      Object.assign(roadmaps.value[roadmapIdx], roadmap);
```

Comment:
Same as [this](https://github.com/logchimp/logchimp/pull/1447/files#r2506840990)